android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BluetoothSite.java

```diff
@@ -54,18 +54,25 @@
   private ProgressDialog progress;
   private int transferRate = 20;
   private StringBuilder html;
+  private Boolean running = false;
```

Comment:
I'd like to blame an overly ambitious shift key.  But yeah, it could be a primitive.